packages/browser-extension-signing-manager/src/lib/browser-extension-signing-manager.ts

```diff
@@ -66,6 +53,20 @@ export class BrowserExtensionSigningManager implements SigningManager {
     return accounts.map(({ address }) => changeAddressFormat(address, ss58Format));
   }
 
+  /**
+   * Return the addresses of all Accounts along with other metadata in the Browser Wallet Extension
+   *
+   * @throws if called before calling `setSs58Format`. Normally, `setSs58Format` will be called by the SDK when instantiated
+   */
+  public async getAccountsWithMeta(): Promise<InjectedAccountWithMeta[]> {
```

Comment:
```suggestion
  public async getAccountsWithMetadata(): Promise<InjectedAccountWithMeta[]> {
```
(style) Probably personal preference, but I think the slightly longer name looks better for a public method.